src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_pending.properties

```
@@ -0,0 +1,38 @@
+pendingScripts=Scripts pending approval
+
+# pending scripts
+## table headers
+pendingHash=Hash
+pendingHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
+pendingExpandCode=Code
+pendingLanguage=Language
+pendingScriptLength=Length
+pendingScriptLength_tooltip=Length of the script in characters
+contextUser=Requester
+contextItem=Context
+requestApprovalTime=Request date
+requestApprovalTime_tooltip=Inform about when the approval request was done.
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script does not provide the user information.
+noRequestApprovalTimeSinceMetadata=N/A
+noRequestApprovalTimeSinceMetadata_tooltip=It was requested before the introduction of metadata.
+noContextItemSinceMetadata=N/A
+noContextItemSinceMetadata_tooltip=The context item was not provided by the code asking for the approval.
+
+approvePending=Approve all selected
+approvePending_tooltip=Approving a pending script will allow any user to execute it. You need to be careful to only approve non-dangerous scripts.
+approvePending_confirm=Really approve all selected scripts? Have you checked that none of them are dangerous?
+denyPending=Deny all selected
+denyPending_tooltip=Denying a pending script will just remove it from the pending list. If there are new script execution attempts, it will reappear in this list.
+denyPending_confirm=Really deny all selected scripts? If they are triggered again, they will reappear in this list.
+pendingNoSelectedItemError=Please select at least one line.
+
+pendingExpand=expand code
+pendingExpand_tooltip=Expand the source code visualization
+pendingMinimize=minimize code
+pendingMinimize_tooltip=Minimize the source code visualization
+
+noPendingScripts=No pending script approvals.
```

```suggestion
noPendingScripts=No scripts pending approval.
```
This is my prefered word order.